site/src/routes/embedding-knn/AgreementClusterStaticPlot.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  export let data: any[] = [];
+
+  var names = data.map((d) => d["ego_primary_label"]);
```

Comment:
Svelte has a `$` operator -- if we declare variable this way, the variable will get updated every time any variable it references changes.

```suggestion
 $: names = data.map((d) => d["ego_primary_label"]);
```